include/seqan3/alphabet/aminoacid/aa27.hpp

```diff
@@ -31,28 +31,33 @@
 // DAMAGE.
 //
 // ============================================================================
-// Author: Sara Hetzel <sara.hetzel AT fu-berlin.de>
-// ============================================================================
+
+/*!\file aa27.hpp
+ * \ingroup aminoacid
+ * \author Sara Hetzel <sara.hetzel AT fu-berlin.de>
+ * \brief Contains seqan3::aa27, container aliases and string literals.
+ */
 
 #pragma once
 
 #include <cassert>
 
 #include <seqan3/alphabet/concept.hpp>
```

Comment:
The alphabet_test fails if I move this down to the NDEBUG section. Does the test needs this to be included?